### PR TITLE
D2K - Add Campaign Tooltip

### PR DESCRIPTION
**Pull request by MustaphaTR**
Depends on #14404.

### CODE REVIEW

**Comment by pchote** [comment]
Dependency has been merged, so can now be rebased to help with testing.

**Comment by MustaphaTR** [comment]
Rebased.

**Review by pchote — DISMISSED** [review]

File: mods/d2k/rules/campaign-tooltips.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+		GenericVisibility: Enemy, Neutral
+		NeutralPrefix: Neutral
+		ShowOwnerRow: false
+
+^Husk:
+	Tooltip:
+		GenericVisibility: Enemy, Ally, Neutral
+		ShowOwnerRow: false
+		AllyPrefix:
+		EnemyPrefix:
```
Comment (abcdefg30) [inline mods/d2k/rules/campaign-tooltips.yaml]: `GenericStancePrefix: false` can (and should imho) be used instead.

Comment (MustaphaTR) [reply mods/d2k/rules/campaign-tooltips.yaml]: Yup, looks like i forgot to update this PR after changing the Neutral Prefix PR.

**Comment by MustaphaTR** [comment]
Updated.

**Review by abcdefg30 — APPROVED** [review]
👎 I really oppose the idea to hide the owner row for good. It just hides the "hacks" going on behind the veil. Ideally we should rather fix that 'properly'. However, I see that this is not going to happen (soon) and considering that several missions were already merged with that, I fear I need to take this in.

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/87128314693a2d6c234cd42a9e5498842501e92d...0e0c4d8dc3650a761f202a0afed5751e54f2712d)

**Pull request merged**